spans_buffer.go

```diff
@@ -1,57 +1,74 @@
 package lightstep
 
-import "github.com/opentracing/basictracer-go"
+import (
+	"time"
 
-const defaultMaxSpans = 1000
+	"github.com/opentracing/basictracer-go"
+)
 
 type spansBuffer struct {
-	rawSpans      []basictracer.RawSpan
-	maxBufferSize int
+	rawSpans    []basictracer.RawSpan
```

Comment:
Total nit, but since you are here, I might call this `report` (or `partialReport`) rather than `spansBuffer`.